ntbs-service/Pages/Notifications/Edit/_ClinicalDetailsTreatmentOptions.cshtml

```diff
@@ -53,7 +53,7 @@
         </nhs-fieldset>
     </nhs-form-group>
 
-    <nhs-form-group nhs-form-group-type="Standard">
+    <nhs-form-group nhs-form-group-type="Standard" id="ClinicalDetails-ECM">
```

Comment:
Why is this ID needed?

Reply:
Not needed, I was going to use it as an anchor for the alert link but that isn't needed.
This fits in with the rest of the page now though.

Reply:
Ok, that's fine